ferrors/error.go

```diff
@@ -518,6 +518,32 @@ func NewAlreadyExistsError(msg string, fields ...Field) Ferror {
 	}
 }
 
+// NewInternalError returns an internal error.
+// It also records the stack trace at the point it was called.
+func NewInternalError(msg string, fields ...Field) Ferror {
+	return &withFields{
+		fundamental: &fundamental{
+			ErrorCode: Internal,
+			stack:     callers(),
+			Msg:       msg,
+		},
+		Fields: fields,
+	}
```

Comment:
We don't need to use `withField` here as we are not going to send which field caused the error.
Just returning the fundamental would be suffice.